Package.swift

```diff
@@ -67,7 +67,8 @@ let package = Package(
                "models",
                "tests",
                "CMakeLists.txt",
-               "Makefile"
+               "Makefile",
+               "ggml/src/ggml-metal-embed.metal"
```

Comment:
I think there might be a similar issue with the cmake build. I think something like this might work for cmake if you'd like to add it:
```console
diff --git a/Package.swift b/Package.swift
index d246658b..8a064fd8 100644
--- a/Package.swift
+++ b/Package.swift
@@ -68,7 +68,9 @@ let package = Package(
                "tests",
                "CMakeLists.txt",
                "Makefile",
-               "ggml/src/ggml-metal-embed.metal"
+               "ggml/src/ggml-metal-embed.metal",
+               "./build/autogenerated/ggml-metal-embed.metal",
+               "./build/bin/ggml-metal.metal"
             ],
             sources: sources,
             resources: resources,
```